src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -92,7 +99,8 @@ HighlightingPage::HighlightingPage()
                 // Case-sensitivity doesn't make sense for user names so it is
                 // set to "false" by default & no checkbox is shown
                 view->setTitles({"Username", "Flash\ntaskbar", "Play\nsound",
```

Comment:
Makes sense to make an additional comment/modify the existing comment now that "Case-sensitivity" is part of the titles here, despite it not being rendered

Reply:
Hope I understood this correctly. If so, resolved in 18a3b599.